app/views/admin/users/edit.html.haml

```diff
@@ -24,84 +22,129 @@
             Email: Unstolen
           %td
             = "&#x2713;".html_safe if @user.notification_unstolen
-= form_for [:admin, @user] do |f|
-  .row-fluid
-    - if @user.errors.any?
-      #alert-block
-        .alert.alert-error
-          %h4
-            Please fix the following
-            = pluralize(@user.errors.count, "error")
-          %ul
-            - @user.errors.full_messages.each do |msg|
-              %li
-                = msg
-    .span6.padded
-      .form.form-horizontal
-        %h3
-          User attributes
-        .control-group.padded
-          %lable.control-label
+        %tr
+          %td
             Created
-          .controls.convertTime
+          %td.convertTime
```

Comment:
Nice! I hadn't tried putting the convertTime on `td`s, it makes sense and simplifies things though!